src/constants/tooltips.ts

```diff
@@ -69,3 +69,4 @@ export const KustomizeTabTooltip = `View Kustomizations (${KEY_CTRL_CMD}+SHIFT+K
 export const HelmTabTooltip = `View Helm Charts (${KEY_CTRL_CMD}+SHIFT+H)`;
 export const TemplatesTabTooltip = `View Templates`;
 export const ValidationTabTooltip = `View Validation (${KEY_CTRL_CMD}+SHIFT+V)`;
+export const ResetFiltersTooltip = `Reset Filters (${KEY_CTRL_CMD}+ALT+R)`;
```

Comment:
is this a standard tooltip to reset filters or a search query?

Reply:
It's just a tooltip for the reset filters icon